schema/crdb/6.1.0/up1.sql

```diff
@@ -0,0 +1 @@
+CREATE SEQUENCE IF NOT EXISTS omicron.public.ipv4_nat_version START 1 INCREMENT 1;
```

Comment:
Couple nitpicks on versioning:

1. We've had other breaking changes, the latest is `8.0.0`, so this should be higher than that
2. This should probably just be `9.0.0` (or whatever that largest major version is at the time of merging)

https://github.com/oxidecomputer/omicron/blob/ebb25ebbb27892cbfe8bb2d00f6cf7b8de563a29/schema/crdb/README.adoc?plain=1#L56

This is semver because we will want to have version changes that aren't breaking in the future, but **right now**, we aren't making guarantees about backward compatibility. To that end, all changes are basically breaking changes.

Reply:
This versioning was from my workaround to the previous issue we encountered regarding the anonymous constraints in the db schema, I can bump this to a proper major version number now that I have the fix (or will have the fix) in this branch.